assets/js/components/Spinner.jsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
-import { computedIconCssClass } from '@lib/icon';
-
 import { EOS_LOADING_ANIMATED } from 'eos-icons-react';
 
-function Spinner({ centered = false }) {
+function Spinner({
+  centered = false,
+  wrapperClassName = '',
+  size = 'm',
+  spinnerColor = 'fill-jungle-green-500',
```

Comment:
When do we need a custom color? :thinking:

Reply:
Agree